src/main/webapp/styles/dashboard.css

```diff
@@ -17,6 +17,29 @@ body {
     flex-direction: row;
 }
 
+.header {
+    align-items: flex-end;
+    display: flex;
+    justify-content: flex-end;
+    margin: 0.5em;
+}
+
+.btn {
+    background-color: #fff;
+    border: none;
+    border-bottom: 2px solid #78938A;
+    font-family: "Montserrat";
+    font-size: 0.8em;
+    margin-right: 2em;
+}
+
+button.btn.create-btn {
+    background-color: #78938A;
```

Comment:
it would be good if the theme colors are set as global variables at root, then it'd be clear to see which color is being used. You can have look at changes in script.css of "Create Trip" PR.